scientific_library/tvb/datatypes/cortex.py

```diff
@@ -52,7 +52,7 @@ class Cortex(HasTraits):
 
     region_mapping_data = Attr(
         field_type=region_mapping.RegionMapping,
-        label="region mapping",
+        label="Region mapping",
```

Comment:
No big deal to change this here, but I think it will be safer if we capitalize all labels when rendering the Jinja template.

Reply:
Okay, I will keep that in mind. I capitalized it because every other parameter from the Simulation Controller is written with capital letters.